src/wallet/rpcwallet.cpp

```diff
@@ -3422,17 +3431,34 @@ static UniValue bumpfee(const JSONRPCRequest& request)
         }
     }
 
-    // sign bumped transaction
-    if (!feebumper::SignTransaction(*pwallet, mtx)) {
-        throw JSONRPCError(RPC_WALLET_ERROR, "Can't sign transaction.");
-    }
-    // commit the bumped transaction
-    uint256 txid;
-    if (feebumper::CommitTransaction(*pwallet, hash, std::move(mtx), errors, txid) != feebumper::Result::OK) {
-        throw JSONRPCError(RPC_WALLET_ERROR, errors[0]);
-    }
     UniValue result(UniValue::VOBJ);
-    result.pushKV("txid", txid.GetHex());
+
+    // sign and commit the bumped transaction if requested or just return the psbt
+    if (add_to_wallet) {
+
+        if (!feebumper::SignTransaction(*pwallet, mtx)) {
+            throw JSONRPCError(RPC_WALLET_ERROR, "Can't sign transaction.");
+        }
+
+        uint256 txid;
+        if (feebumper::CommitTransaction(*pwallet, hash, std::move(mtx), errors, txid) != feebumper::Result::OK) {
+            throw JSONRPCError(RPC_WALLET_ERROR, errors[0]);
+        }
+        result.pushKV("txid", txid.GetHex());
+    } else {
+        // Return a as filled-out-as-possible PSBT
+        PartiallySignedTransaction psbtx(mtx);
+        bool complete = false;
+        const TransactionError err = FillPSBT(pwallet, psbtx, complete, SIGHASH_ALL, true /* sign */, true /* bip32derivs */);
```

Comment:
nit: I find it more readable to have the variable name before the value (as it would be for named params)

Reply:
Other psbt methods do not set `bip32derivs` to `true` by default (though I'm starting to regret [recommending that](https://github.com/bitcoin/bitcoin/pull/13557#pullrequestreview-135905054)). To be consistent it should be an option.

Reply:
See #17264

Reply:
> I find it more readable to have the variable name before the value

Following convention here

> To be consistent it should be an option

IMO nah, especially after I nuke all the additional arguments